client/src/main/java/io/narayana/lra/client/internal/NarayanaLRAClient.java

```diff
@@ -181,6 +177,7 @@ public NarayanaLRAClient() {
      * @param port port where the LRA coordinator will be contacted
      * @param coordinatorPath path where the LRA coordinator will be contacted
      */
+    @Deprecated(since = "1.0.3.Final", forRemoval = true)
```

Comment:
Not sure about this, see issue https://issues.redhat.com/browse/JBTM-3940
"Now that the [client api](https://github.com/jbosstm/narayana/blob/7.1.0.Final/rts/lra/client/src/main/java/io/narayana/lra/client/internal/NarayanaLRAClient.java#L93) has been moved to an internal package the @Deprecated annotation can be removed."

Reply:
It's in the internal package so we don't actually need to deprecate it IMO. But we can notify developers on Zulip of the change if we want?

Reply:
> unless we can move to JPMS :)

Yes that is the plan.